tests/default_vars.sh

```diff
@@ -268,7 +268,64 @@ elif [[ $MACHINE_ID = expanse.* ]]; then
 
   TPN_cpl_atmw_gdas=12; INPES_cpl_atmw_gdas=6; JNPES_cpl_atmw_gdas=8
   THRD_cpl_atmw_gdas=2; WPG_cpl_atmw_gdas=24; APB_cpl_atmw_gdas="0 311"; WPB_cpl_atmw_gdas="312 559"
+  
+elif [[ $MACHINE_ID = AWS_cloud.* ]]; then
```

Comment:
@SadeghTabas-NOAA @zach1221 Why are these platform-specific instead of just `noaacloud.*`? I get an UNKNOWN PLATFORM error. Are they intended to be used as templates that users can copy into a `noaacloud.*` block?

Reply:
@gspetro-NOAA default_vars still needs work. I'm not finished making edits to it.